src/UIAutomation/Microsoft.UI.UIAutomation/Client.g.cpp

```diff
@@ -489,6 +489,37 @@ namespace winrt::Microsoft::UI::UIAutomation::implementation
         return result;
     }
 
+    AutomationRemoteMetadata::AutomationRemoteMetadata(bytecode::OperandId operandId, AutomationRemoteOperation& parent)
+        : base_type(operandId, parent)
+    {
+    }
+
+    void AutomationRemoteMetadata::Set(const AutomationRemoteMetadata::class_type& rhs)
+    {
+        AutomationRemoteObject::Set<AutomationRemoteMetadata>(rhs);
+    }
+
+    winrt::AutomationRemoteBool AutomationRemoteMetadata::IsEqual(const AutomationRemoteMetadata::class_type& rhs)
+    {
+        return AutomationRemoteObject::IsEqual<AutomationRemoteMetadata>(rhs);
+    }
+
+    winrt::AutomationRemoteBool AutomationRemoteMetadata::IsNotEqual(const AutomationRemoteMetadata::class_type& rhs)
+    {
+        return AutomationRemoteObject::IsNotEqual<AutomationRemoteMetadata>(rhs);
+    }
+
+    winrt::AutomationRemoteMetadata AutomationRemoteOperation::NewEnum(AutomationMetadata initialValue)
+    {
+        const auto resultId = GetNextId();
+        InsertInstruction(bytecode::NewInt{
+            resultId,
+            static_cast<int>(initialValue)
+            });

```

Comment:
Seems like your editor decided to show who is the boss and extra-indented the closing parenthesis.